bin/lint-markdown-api-history.ts

```diff
@@ -237,6 +247,34 @@ async function main(
           }
         }
 
+        // Throw an error if a description isn't surrounded by double quotation marks
+        if (checkDescriptions) {
+          const possibleDescription = codeBlock.value.matchAll(possibleDescriptionRegex);
+
+          for (const [matchedLine, matchedGroup] of possibleDescription) {
+            const trimmedMatchedGroup = matchedGroup.trim();
+            const isMatchedGroupInsideQuotes =
+              trimmedMatchedGroup.startsWith('"') && trimmedMatchedGroup.endsWith('"');
+
+            if (isMatchedGroupInsideQuotes) continue;
+
+            console.error(
+              'Error occurred while parsing Markdown document:\n\n' +
+                `'${filepath}'\n\n` +
+                'Possible description field is not surrounded by double quotes.\n\n' +
+                'This might cause issues when parsing the YAML (might not throw an error)\n\n' +
+                'Matched group:\n\n' +
+                `${matchedGroup}\n\n` +
+                'Matched line:\n\n' +
+                `${matchedLine}\n\n` +
+                'API history block:\n\n' +
+                `${possibleHistoryBlock.value}\n`,
+            );
+            errorCounter++;
+            continue;
```

Comment:
The rare occasion that a labeled statement is useful. 😅 Normally I'd steer away from them just because in general labels make code flow harder to follow, but I think since this is an isolated case it's acceptable.

I think you want to call it `historyBlockForLoop:` and put it on line 172/182, though? You just want to skip that one block, not the whole document, to be consistent with the other error handling.

Also, I think it'll read a little better if you restructure it to not do two different variations of `continue` in the same loop:

```suggestion
            if (!isMatchedGroupInsideQuotes) {
              console.error(
                'Error occurred while parsing Markdown document:\n\n' +
                  `'${filepath}'\n\n` +
                  'Possible description field is not surrounded by double quotes.\n\n' +
                  'This might cause issues when parsing the YAML (might not throw an error)\n\n' +
                  'Matched group:\n\n' +
                  `${matchedGroup}\n\n` +
                  'Matched line:\n\n' +
                  `${matchedLine}\n\n` +
                  'API history block:\n\n' +
                  `${possibleHistoryBlock.value}\n`,
              );
              errorCounter++;
              continue historyBlockForLoop;
            }
```

Reply:
Whoops, wrote that right before going to bed 😅. _Hopefully_ (🙏), this is correct [`7fa70bd` (#80)](https://github.com/electron/lint-roller/pull/80/commits/7fa70bde7f8439998e365e673a3d7c057ade33df)